featuretools/utils/trie.py

```diff
@@ -0,0 +1,116 @@
+class Trie(object):
+    """
+    A trie (prefix tree) where the keys are sequences of hashable objects.
+
+    It behaves similarly to a dictionary, except that the keys can be lists or
+    other sequences.
+
+    Examples:
+        .. code-block:: python
```

Comment:
i believe this doc string needs to be updated using `.value`

Reply:
Is there a way to have this block executed when the docs are built?

Reply:
yep, if you want it to be a test case, look at the syntax we use for any of the primitives' example sections. In that case, building the docs will run the code and error if the outputs don't match. 

if you want to just run it, look at how we do `encode_features`